docs/rackhd/authentication.rst

```diff
@@ -0,0 +1,174 @@
+Accessing RackHD APIs with Authentication
+-----------------------------------------
+
+With authentication is enabled in config.json configure file for an endpoint,
+see `Setting up HTTP/HTTPS endpoint`_ and `Authentication`_, authentication is needed for
+accessing the API group associated with the specified endpoint.
+
+This section describes how to access APIs that need authentication.
+
+Setup endpoints to enable authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Please refer to `Setting up HTTP/HTTPS endpoint`_ on how to setup endpoints. Simply put,
```

Comment:
Thanks @benbp and @yyscamper . I do intended to leave a pause here, this will break the line shorter.